src/constants/constants.ts

```diff
@@ -90,7 +90,8 @@ export const ROUTES = {
   DATA_REGULATION: '/data-regulation',
   LOGOUT: '/logout',
   HUB: '/hub?SkipWelcomeScreen=true',
-  ELIGIBILITY: '/hub/eligibility',
+  GHMD247: '/hub/eligibility', // GHMD-247: disable direct link to eligibility screen from mobile app
+  ELIGIBILITY: '/hub/eligibility-steps',
```

Comment:
This allows people like Solly to set ?forceShowEnrollment=true and step through the enrollment, and essentially retire the old /hub/eligibility route (point it to the main Hub)